default.json

```diff
@@ -93,6 +93,7 @@
         "^@?seek",
         "seek$",
         "^@aws-sdk/",
+        "^@smithy/",
```

Comment:
🤔 This rule is scoped to `devDependencies; these exclusions represent the packages that you want to exclude from the consolidated `deps: npm dev dependencies` PR (i.e. because they are grouped/PRed separately). Is smithy a prod or dev dep in your example?

Reply:
Hmmm it's a prod dep. Doesn't completely bother me but I thought it would always be grouped.

https://github.com/SEEK-Jobs/hirer-products-api/blob/73c21b676d75b3856945dabab3dcb6340d207d2b/package.json#L55